### PR TITLE
implementation Seeds

### PR DESCRIPTION
**Pull request by Pub4Game**

### CODE REVIEW

**Comment by Pub4Game** [comment]
Ohh... Conflict

**Comment by Pub4Game** [comment]
:cry:

**Comment by Angelic47** [comment]
Conflicts...

**Comment by Pub4Game** [comment]
@Angelic47 New pull request?

File: src/main/java/cn/nukkit/block/Block.java [hunk 1]
```diff
@@ -383,10 +383,10 @@ public static void init() {
             list[IRON_BARS] = IronBars.class;
             list[GLASS_PANE] = GlassPane.class;
             list[MELON_BLOCK] = Melon.class;
-            /*list[PUMPKIN_STEM] = PumpkinStem.class;
+            list[PUMPKIN_STEM] = PumpkinStem.class;
             list[MELON_STEM] = MelonStem.class;
-            list[VINE] = Vine.class;
-            */
+            //list[VINE] = Vine.class;
```
Comment (Angelic47) [inline src/main/java/cn/nukkit/block/Block.java]: Oops, vine is implemented but here disabled it again..

**Comment by Angelic47** [comment]
@Pub4Game OK, just a new pr :D

**Pull request closed without merging**